### DISCUSSION TITLE
provisioner is not coming

### DISCUSSION THREAD

**Discussion by prabalark** [learndevopsonline/learndevopsonline#1916] [Q&A]
ip - 34.239.123.213
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/7922c305-8073-426a-a8fa-1cbaf10a4f03)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/32f866f6-b438-4e22-8c94-d71b24cca1dc)

**Comment by chaithanya1812** [comment]
Hi @prabalark ,
## provisioner block has be inside of resource block.

**Comment by prabalark** [comment]
ok got it

**Comment by prabalark** [comment]
getting below error
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/8cac8bc4-15c4-45ef-9eff-dab1ca7eff94)

**Comment by chaithanya1812** [comment]
Add a provider block to your code

**Comment by prabalark** [comment]
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/be14b89d-2a87-43fd-867d-75f64685cc29)
![image](https://github.com/learndevopsonline/learndevopsonline/assets/130270906/7f813496-cd80-489e-bacb-ab5c02134f80)

**Comment by chaithanya1812** [comment]
## Do we have resource called var_resource ??
![image](https://github.com/learndevopsonline/learndevopsonline/assets/111736742/73022698-a1f3-4dc8-9369-e1e3fc03c513)

**Comment by prabalark** [comment]
ok changed and it working ,
 thanks
